src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -129,7 +129,7 @@ <h2 class="card-pf-title">
         <button class="btn btn-primary"
                 [disabled]="stepCompleted !== true"
                 (click)="navToNextStep()">Continue</button>
-        <button class="btn btn-link" (click)="launcherComponent.cancel()">Cancel</button>
+        <button class="btn btn-default" (click)="launcherComponent.cancel()">Cancel</button>
```

Comment:
Please change back to `.btn-link`, per design

Reply:
@mindreeper2420 Hi Adam. We are following this UX link for design reference - https://redhat.invisionapp.com/share/6BEZL73M7#/screens/275860134

According to this design, 'Cancel' appears in the button format.

If you have any other design reference for this, please share it so that we can follow them as well.

Reply:
@arunkumars08 you are correct, in that is the proper design. We've been doing them as button links during peer-review with Brian, so I will verify with him if that is an older design for the buttons, or what currently exists (btn-link) is incorrect.

Thank you for checking and looking at that. I've been staring at this for so long, I didn't even notice the difference 😃

Reply:
@mindreeper2420 Oh got it. No problem. So, I am leaving my current change as is.